glue_vispy_viewers/common/vispy_widget.py

```diff
@@ -108,6 +126,9 @@ def _update_limits(self):
     def _reset_view(self):
         self.view.camera.reset()
 
+    def rotate(self, event):
+        self.rotate_speed += self.rotate_slider_val
+        self.view.camera.azimuth = self.rotate_speed
```

Comment:
Here I wonder whether it wouldn't make more snese to just do:

```
self.view.camera.azimuth += self.rotate_slider_val
```

(otherwise it sounds like the speed is increasing all the time)